docs/misc.rst

```diff
@@ -0,0 +1,203 @@
+
+
+Notes
+-----
+
+Timing & The Microsoft Azure cloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Timing is a tricky subject when dealing with the Azure cloud as
```

Comment:
Timing is a **complex** subject when dealing with the Azure cloud ~~as~~ **because**